lib/galaxy/queue_worker.py

```diff
@@ -183,6 +183,9 @@ def _get_new_toolbox(app, save_integrated_tool_panel=True):
     app.datatypes_registry.load_datatype_converters(new_toolbox, use_cached=True)
     app.datatypes_registry.load_external_metadata_tool(new_toolbox)
     load_lib_tools(new_toolbox)
+    # Load built-in converters
+    if app.config.display_builtin_converters:
+        new_toolbox.load_builtin_converters()
```

Comment:
Should this be part of `ToolBox.__init__()` instead of being duplicated in `lib/galaxy/app.py` and here?

Reply:
I also thought so. How about leaving it as it is for the bugfix and I open a follow-up on dev once this is merged forward.

Reply:
But I can also do it here. As you prefer.

Reply:
Since this will need to be merged forward a couple of releases, I'd rather have it in the right place from the start.
Obviously, it that actually works :D

Reply:
Let's see if there is any fallout in the tests (would it make sense to open a PR against dev to check against recent tests)?

I can then also test this on my instance.